docs/source/objects.rst

```diff
@@ -2,13 +2,8 @@
 Objects
 =======
 
-Module: :mod:`kubespawner.objects`
-----------------------------------
-
 .. automodule:: kubespawner.objects
 
-.. currentmodule:: kubespawner.objects
-
```

Comment:
AHA! Have this perhaps caused documentation duplications in the past? I remember such bug.

Reply:
I think this was harmless, it's just duplicated in automodule as well.